### PR TITLE
Improve responsiveness and add changelog management

### PR DESCRIPTION
**Pull request by Dani21**
Requires API change to enable

### CODE REVIEW

**Review by corywright — APPROVED** [review]
Will wait for @paulkva's review before merging.

**Comment by Dani21** [comment]
Had a couple of bug fixes to submit, a regular expression was breaking the site for older browsers and site save messages weren't being handled anymore. I also changed all form submission messages to use [bootstrap alerts](https://getbootstrap.com/docs/3.4/components/#alerts) instead of red/green text. Changes redeployed to test.

**Pull request merged**
